### PR TITLE
Add 8ball

### PR DESCRIPTION
**Pull request by godev111222333**

### CODE REVIEW

File: cosmos/8ball.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      "coinDenom": "EBL",
+      "coinMinimalDenom": "uebl",
+      "coinDecimals": 6,
+      "coinGeckoId": "8ball"
+    },
+    {
+      "coinDenom": "EBL",
+      "coinMinimalDenom": "uebl",
+      "coinDecimals": 6
+    }
```
Comment (HeesungB) [inline cosmos/8ball.json:39]: @godev111222333 Duplicate token entered, Could you remove one?

Comment (godev111222333) [reply cosmos/8ball.json]: Removed, thank you!

**Comment by HeesungB** [comment]
@godev111222333  Your chain identifier is `eightball`, so You have to change file name `cosmos/eightball.json` and `images/eightball/chian.png`

**Comment by HeesungB** [comment]
@godev111222333 
Failed to connect websocket to /websocket rpc
Is there websocket in RPC server?

File: cosmos/eightball.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
+  "chainId": "eightball-1",
+  "chainName": "8ball",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/8ball/chain.png",
```
Comment (HeesungB) [inline cosmos/eightball.json:4]: Could you change `chainSymbolImageUrl` field?

"https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/8ball/chain.png"
8ball -> eightball

**Comment by HeesungB** [comment]
@godev111222333 Thank you for contribution!

**Pull request merged**